packages/hackathon/src/authToken/extensionProxyTransport.ts

```diff
@@ -57,7 +56,7 @@ export class ExtensionProxyTransport extends BaseTransport {
     authenticator?: Authenticator,
     options?: Partial<ITransportSettings>
   ) {
-    const agentTag = options?.agentTag || `${agentPrefix} ${lookerVersion}`
+    const agentTag = options?.agentTag || agentPrefix
```

Comment:
does `options?.agentTag` already have the `lookerVersion renamed to sdkVersion` info embedded? Or are we ditching that piece from the agentTag?

Reply:
the extension proxy agentTag should be initialized at this point. If it isn't, it defaults.